microservice-transaction-sample/README.md

```diff
@@ -1,225 +1,183 @@
-*NOTE: We also plan to create a microservice container for the coordinator database to truly achieve the database-per-service pattern.*
+As shown in the diagram, both services access a small coordinator database used for the Consensus Commit protocol. The database is service-independent and exists for managing transaction metadata for Consensus Commit in a highly available manner.
 
-In this sample application, for ease of setup and explanation, we co-locate the coordinator database in the same Cassandra instance of the Order Service, but of course, the coordinator database can be managed as a separate database.
+In the sample application, for ease of setup and explanation, we co-locate the coordinator database in the same Cassandra instance of the Order Service. Alternatively, you can manage the coordinator database as a separate database.
 
-Also, note that application-specific error handling, authentication processing, etc., are omitted in the sample application since it focuses on explaining how to use ScalarDB.
-Please see [this document](https://github.com/scalar-labs/scalardb/blob/master/docs/api-guide.md#handle-exceptions) for the details of how to handle exceptions in ScalarDB.
+{% capture notice--info %}
+**Note**
 
-Additionally, you assume each service has one container in this sample application to avoid considering request routing between the services.
-However, for production, because each service typically has multiple servers (or hosts) for scalability and availability, you should consider request routing between the services in Two-phase Commit Transactions.
-Please see [this document](https://github.com/scalar-labs/scalardb/blob/master/docs/two-phase-commit-transactions.md#request-routing-in-two-phase-commit-transactions) for the details of Request Routing in Two-phase Commit Transactions.
+Since the focus of the sample application is to demonstrate using ScalarDB, application-specific error handling, authentication processing, and similar functions are not included in the sample application. For details about exception handling in ScalarDB, see [Handle exceptions](https://github.com/scalar-labs/scalardb/blob/master/docs/api-guide.md#handle-exceptions).
 
-### Schema
+Additionally, for the purpose of the sample application, each service has one container so that you can avoid using request routing between the services. However, for production use, because each service typically has multiple servers or hosts for scalability and availability, you should consider request routing between the services in transactions with a two-phase commit interface. For details about request routing, see [Request routing in transactions with a two-phase commit interface](https://github.com/scalar-labs/scalardb/blob/master/docs/two-phase-commit-transactions.md#request-routing-in-transactions-with-a-two-phase-commit-interface).
```

Comment:
The link does not direct us to the `[Request routing in Two-phase Commit Transactions](https://github.com/scalar-labs/scalardb/blob/master/docs/two-phase-commit-transactions.md#request-routing-in-two-phase-commit-transactions)` section, but is it resolved later?
Also, I think we can refer to ScalarDB Cluster here because the cluster is the one that does such routing.

Just a heads-up, but the referenced page says [Two-phase Commit Transactions](https://github.com/scalar-labs/scalardb/blob/master/docs/two-phase-commit-transactions.md#two-phase-commit-transactions), so it should also be revised based on the naming rule.

Reply:
Correct, this will be resolved later in another PR that is being reviewed. 

Since the changes that we made to the naming in [`scalardb` PR #1082](https://github.com/scalar-labs/scalardb/pull/1082/commits/98c72af54452c475987db2281fe95229a5953906) (now in [`scalardb` PR #1117](https://github.com/scalar-labs/scalardb/pull/1117)) have not been merged yet, this link doesn't work. 

Before merging this PR for the samples, we should merge the changes related to the updated `Transactions with a Two-Phase Commit Interface` doc across all necessary version branches so that we don't have broken links in this doc.

Reply:
OK, thank you for the clarification!